test/formal/combine_shr_shl_by_constant_64.py

```diff
@@ -33,7 +33,7 @@
 Mask = SHL(B, SHR(A, Int2BV(IntVal(-1), n_bits)))
 opt = If(
 	UGT(A, B),
-	AND(SHR(a - b, x), Mask),
```

Comment:
Maybe we should really add a CI run, so that we notice stuff like this ;-).

Reply:
yea we should..